lesson-4/script.js

```diff
@@ -0,0 +1,74 @@
+const correctResult = (0.1 + 0.2).toFixed(1);
+console.log(correctResult);
+const userLogin = prompt("Enter your login.");
+console.log(userLogin);
+const userPassword = prompt("Enter your password.");
+console.log(userPassword);
+if (userLogin == "admin" && userPassword == "12pass33210") {
+    console.log("Access is allowed.");
+} else if (userLogin != "admin" && userPassword != "12pass33210") {
+    console.log("This user have no access to the panel.");
+} else {
+    console.log("Login or password are wrong.")
+}
+let userMoney = +prompt("How much money do you have?");
+console.log(`User have ${userMoney} UAH`);
+console.log("Here is our price list below:")
+let Watermelon = 30;
+console.log(`Watermelon: ${Watermelon} UAH`);
+let Carrot = 8;
```

Comment:
```suggestion
const carrot = 8;
```
and so on. Prices in this caser are not changing. And naming variables with uppercase - is not recommended, will discuss why later

Reply:
so, we must use const every time in such local cases, witch are separated of whole project - OK. Capitalized - my mistake, I missed it

Reply:
Yes, `const` as much as possible